hypervisor/common/sched_prio.c

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (C) 2021 Intel Corporation. All rights reserved.
```

Comment:
@dbkinder , isn't the recommendation to **not** include the `All rights reserved.` piece?

Reply:
Yes, remove "All rights reserved".

Reply:
I will remove "All rights reserved".